### DISCUSSION TITLE
v3.5.0版本

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#395] [Announcements]
## What's Changed
* fix: 下一页修复右侧没有箭头 by @mouyase in https://github.com/tangly1024/NotionNext/pull/379
* 支持配置Google字体地址 by @tangly1024 in https://github.com/tangly1024/NotionNext/pull/384
* Feature/index button category by @tangly1024 in https://github.com/tangly1024/NotionNext/pull/393
* Bugfix/list style type lower alpha by @tangly1024 in https://github.com/tangly1024/NotionNext/pull/390
* Feature/custom path by @tangly1024 in https://github.com/tangly1024/NotionNext/pull/392
* doc: correct the Cusdis URL by @haixin1225 in https://github.com/tangly1024/NotionNext/pull/389
* fix: issue when BLOG.SINCE is current year by @haixin1225 in https://github.com/tangly1024/NotionNext/pull/387
* Release/v3.5.0 by @tangly1024 in https://github.com/tangly1024/NotionNext/pull/394

## New Contributors
* @mouyase made their first contribution in https://github.com/tangly1024/NotionNext/pull/379
* @haixin1225 made their first contribution in https://github.com/tangly1024/NotionNext/pull/389

**Full Changelog**: https://github.com/tangly1024/NotionNext/compare/V3.4.5...v3.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tangly1024/NotionNext/releases/tag/v3.5.0'>v3.5.0版本</a>.</em>

**Comment by Leepromax** [comment]
请教一下。
如何可以更直观的看到“分类”，现在需要点左上角菜单，然后点分类，才能看到里面对文章的分类。我想可不可以直接点开左上角菜单直接呈现分类内容呢？


现在page只能是一个单页，后面会不会支持在单页里面套单页呢？
